lib/Exporter/Writer/MimeTypedWriterInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Exporter\Writer;
+
+interface MimeTypedWriterInterface extends WriterInterface
+{
+    /**
+     * @return string the mime type of the output
+     */
+    public function getMimeType();
```

Comment:
Maybe `getDefaultMimeType`?

For example, json has multiple mime types: http://stackoverflow.com/q/477816/1731473